### PR TITLE
authentication.md: improve description of anonymous API requests

### PR DESCRIPTION
**Pull request by amosshapira**
The text uses double-negative which makes it hard to follow what it tries to say.
This change proposes phrasing with better flow, IMHO

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ manually through API calls. Service accounts are tied to a set of credentials
 stored as `Secrets`, which are mounted into pods allowing in-cluster processes
 to talk to the Kubernetes API.
 
-API requests are tied to either a normal user or a service account, or are treated
+API requests not tied to either a normal user or a service account are treated
```
Comment (divya-mohan0209) [inline content/en/docs/reference/access-authn-authz/authentication.md:48]: The gist of what we are trying to convey is that any unauthenticated API requests will be treated as anonymous and they need to be tied to either a normal user presenting a valid certificate signed by the K8s cluster CA/a service account. IMO, I don't think either of the sentences are correct because if a normal user doesn't present a certificate issued by the cluster to the K8s API, the associated request will be treated as an anonymous one. 

WDYT of the suggestion below? 

```suggestion
API requests not tied to a normal user with a valid certificate issued by the cluster's certificate authority or a service account are treated
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/access-authn-authz/authentication.md (hunk 1 above)
Comment (amosshapira) [reply content/en/docs/reference/access-authn-authz/authentication.md:48]: Thanks for the correction.
Is the extra detail about how the client certificate is being verified really critical in this part of the text?
I'm asking as a newcomer who reads this text for the first time - how much is this extra detail important vs the extra complexity of the information?
If you accept what I say, then perhaps:
```suggestion
API requests not tied to either an authenticated normal user or a service account are treated
```

Comment (sftim) [reply content/en/docs/reference/access-authn-authz/authentication.md:48]: > API requests not tied to either an authenticated normal user or a service account are treated

This is a bit tautological. Tying a request to a user _is_ authentication.

Comment (sftim) [reply content/en/docs/reference/access-authn-authz/authentication.md:48]: Also remember that Kubernetes supports either certificate-based authentication (“mutual TLS”), or token based authentication ([TokenReview](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-review-v1/)).

Comment (amosshapira) [reply content/en/docs/reference/access-authn-authz/authentication.md:48]: > Tying a request to a user is authentication.

I got you.
As a relative newcomer to this depth of K8s knowledge, this terminology was confusing because I wasn't familiar with the definition of "tying". Could it be phrased in a way that's less jargon-dependent?

Comment (sftim) [reply content/en/docs/reference/access-authn-authz/authentication.md:48]: This isn't jargon. However, we could use different language that is more accessible for people who don't read English as a first language.

**Comment by tengqm** [comment]
Where are we now? Shall we proceed to merge or close this one?

**Comment by kbhawkey** [comment]
Hi @amosshapira . Is the existing text clear or are you proposing changes?

**Comment by amosshapira** [comment]
> Hi @amosshapira . Is the existing text clear or are you proposing changes?

Yep all good. Thanks.

**Comment by kbhawkey** [comment]
OK, @amosshapira. Can you close your pull request?

`/close`

**Comment by amosshapira** [comment]
> OK, @amosshapira. Can you close your pull request?
> 
> `/close`

Do you mean "merge and close" or just "close"?

**Comment by sftim** [comment]
Perhaps you could clarify what action you'd now like to take, in light of the reviews so far?

**Comment by sftim** [comment]
I'd be happy to see changes come in that are like the ones you suggested @amosshapira
What help can we give you to make that happen?

**Comment by amosshapira** [comment]
I already closed my PR. As someone who was reading the K8s docs for the first time, I found the text confusing. I suggested a clarification and was told that I misunderstood the text. I don't know how to improve it from here.
